doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -18,26 +18,45 @@
 <a name="4.1"></a>
 ## 4.1 Introduction
 
-This chapter describes in detail the Kubernetes Reference Architecture in terms of the functional capabilities and how they relate to the Reference Model requirements, i.e. how the infrastructure profiles are determined, documented and delivered.
-
-The specifications defined in this chapter will be detailed with unique identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g. `ra2.ch.001` for the first requirement in the Container Host section.  These specifications will then be used as requirements input for the Kubernetes Reference Implementation and any Vendor or Community Implementations.
-
-Figure 4-1 below shows the architectural components that are described in the subsequent sections of this chapter.
-
-<p align="center"><img src="../figures/ch04_k8s_architecture.png" alt="Kubernetes Reference Architecture" Title="Kubernetes Reference Architecture" width="65%"/></p>
+This chapter describes in detail the Kubernetes Reference Architecture in terms
+of the functional capabilities and how they relate to the Reference Model
+requirements, i.e. how the infrastructure profiles are determined, documented
+and delivered.
+
+The specifications defined in this chapter will be detailed with unique
+identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g.
+`ra2.ch.001` for the first requirement in the Container Host section.  These
+specifications will then be used as requirements input for the Kubernetes
+Reference Implementation and any Vendor or Community Implementations.
+
+Figure 4-1 below shows the architectural components that are described in the
+subsequent sections of this chapter.
+
+<p align="center"><img src="../figures/ch04_k8s_architecture.png"
+alt="Kubernetes Reference Architecture" Title="Kubernetes Reference
+Architecture" width="65%"/></p>
 <p align="center"><b>Figure 4-1:</b> Kubernetes Reference Architecture</p>
 
 <a name="4.2"></a>
 ## 4.2 Container Host
 
-This section describes the configuration that will be applied to the physical or virtual machine and an installed Operating System.
-In order for a Container Host to be conformant with the Reference Architecture it must be implemented as per the following specifications:
+This section describes the configuration that will be applied to the physical or
+virtual machine and an installed Operating System. In order for a Container Host
+to be conformant with the Reference Architecture it must be implemented as per
+the following specifications:
 
 |Ref|Specification|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.ch.001`||||
-|`ra2.ch.002`||||
-|`ra2.ch.003`||||
+|`ra2.ch.001`|Huge Pages|Huge Pages must be enabled within the Container Host OS, with at least one each of 2048KiB and 1048576KiB page sizes, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.|[infra.com.cfg.004](./chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
```

Comment:
are you trying to say that all containers hosts must have hugepages and SR-IOV enabled all the time, or that it should be possible to enable these features on all hosts?
the former isn't really true, as majority of the applications do not require hugepages, or SR-IOV. if you enable hugepages on a host which does not hold DPDK apps that memory is effectively wasted

it would be better to say these feature must be possible to be configured for any host profiles IMO

Reply:
That's a great point - I will re-word this and other statements.